### PR TITLE
Bump astroid to 2.15.0, update changelog

### PR DESCRIPTION
**Pull request by Pierre-Sassoulas**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -2,10 +2,22 @@
...
+What's New in astroid 2.15.1?
 =============================
 Release date: TBA
 
+
+
+What's New in astroid 2.15.0?
+=============================
+Release date: 2023-03-06
+
```
Comment (Pierre-Sassoulas) [inline ChangeLog:20]: * astroid now support ``TryStar`` nodes from python 3.11.

  Refs #2028

Comment (Pierre-Sassoulas) [inline ChangeLog:20]:
```suggestion

* astroid now support ``TryStar`` nodes from python 3.11 and should be fully compatible with python 3.11.

  Closes #2028 
```

**Comment by DanielNoord** [comment]
Shall we fix all cases of `Uninferable` before releasing `2.17.0` as well?

File: ChangeLog [hunk 2]
```diff
@@ -2,10 +2,25 @@
...
+* astroid now support ``TryStar`` nodes from python 3.11 and should be fully compatible with python 3.11.
+
+  Closes #2028
 * ``Formattedvalue.postinit`` is now keyword only. This is to allow correct typing of the
```
Comment (jacobtylerwalls) [inline ChangeLog:24]:
```suggestion

* ``Formattedvalue.postinit`` is now keyword only. This is to allow correct typing of the
```

**Comment by DanielNoord** [comment]
> Shall we fix all cases of `Uninferable` before releasing `2.17.0` as well?

Never mind. I thought this was `pylint` 😅

File: ChangeLog [hunk 3]
```diff
@@ -2,10 +2,25 @@
...
 =============================
 Release date: TBA
 
+
+
+What's New in astroid 2.15.0?
+=============================
+Release date: 2023-03-06
+
+* astroid now support ``TryStar`` nodes from python 3.11 and should be fully compatible with python 3.11.
```
Comment (jacobtylerwalls) [inline ChangeLog:21]:
```suggestion
* astroid now supports ``TryStar`` nodes from python 3.11 and should be fully compatible with python 3.11.
```

**Review by DanielNoord — APPROVED** [review]

**Pull request merged**